src/platform/bouffalolab/BL602/DeviceInfoProviderImpl.cpp

```diff
@@ -0,0 +1,366 @@
+/*
```

Comment:
Is it a copy paste of `examples/providers/DeviceInfoProviderImpl.cpp`? If so, maybe you could use the latter?

Reply:
If you make changes, you should use your own. otherwise use the examples one as a marker for production applications that they should build their own.

Reply:
I agree with your comments, I have fixed it. I removed the newly added DeviceInfoProviderImpl.cpp and used the default DeviceInfoProviderImpl.cpp. But this triggers a new restyled error. Guess the reason for this problem may be that I deleted the previously restyled file. I can't figure this out, so I'm reposting a pr, 19877.